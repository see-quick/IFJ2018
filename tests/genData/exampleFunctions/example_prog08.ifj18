# Add numbers from 1 to n
    
    a = inputi
    b = 1
    sum = 0
    
	while b <= a do
        sum = sum + b
        b = b + 1
    end
    
    print "\nSum of numbers from 1 to ", a , " is: ", sum, "\n"
