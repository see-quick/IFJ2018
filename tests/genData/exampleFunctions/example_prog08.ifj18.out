5
Sum of numbers from 1 to 5 is: 15
